.github/workflows/golangci-lint.yml

```diff
@@ -16,3 +18,23 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.42
+
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.17.x
+      - name: Release
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Wouldn't this cut a release on every pull request?

Reply:
Maybe yes.

I should probably for the repo and play with this on my own repo to see how it behaves there, how the releases made look like, etc.

Reply:
You might need to remove the push.branches=main and leave only the tags. See this one https://github.com/bitnami-labs/charts-syncer/blob/master/.github/workflows/release.yaml

Reply:
done